web/admin/store/reducers/root.js

```diff
@@ -59,7 +59,7 @@ export const ADMIN_ACTIONS = {
   addUsers: addUsersReducer,
   putAsideOriginalMissions: putAsideOriginalMissionsReducer,
   revertMissionToOriginalValues: revertMissionToOriginalValuesReducer,
-  updateCompanyName: updateCompanyNameReducer
+  updateCompanyNameAndPhoneNumber: updateCompanyNameAndPhoneNumberReducer
```

Comment:
:green_circle: Pour correspondre au naming du call api :
```suggestion
  updateCompanyDetails: updateCompanyDetailsReducer
```

Reply:
Oui alors... il y a déjà un updateCompanyDetailsReducer qui fait complètement autre chose ... Donc j'ai hésité, mais côté back je préférais vraiment l'appeler Name->Details, ici je n'ai pas pu garder ce nom du coup.